calico-cloud_versioned_docs/version-3.15/get-started/connect/cc-arch-diagram.mdx

```diff
@@ -0,0 +1,51 @@
+|                                | **compliance-server** <br />Provides the API for listing, downloading, and rendering reports, and RBAC by performing authentication and authorization through the Kubernetes API server. RBAC is determined from the users RBAC for the Global report type and Global report resources. |                                                              |
+|                                | **compliance-benchmarker**<br />A daemonset that runs checks in the CIS Kubernetes Benchmark on each node so you can see if Kubernetes is securely deployed. |                                                              |
+| Image Assurance                | Identifies vulnerabilities in container images that you deploy to Kubernetes clusters.  Includes these components: <br /><br />**Admission controller**<br />Uses Kubernetes Validating Webhook Configuration https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/ to control which images can be used to create pods based on scan results.  <br /><br />**CLI scanner**<br />A CLI binary tool that scans images. Users can upload the results of a scanned image to the backend IA API.  <br /><br />**Pod watcher** <br />Monitors clusters and collects runtime information about which pods are running which images. <br /> | &bull; TCP 8080 to Guardian<br /> &bull;  TCP 6443 to Kubernetes API server |
+|                                | **API and API proxy**<br />Isolates tenant data and ensures that all external access to Image Assurance data is through the API Proxy, which enforces authorization through Kubernetes RBAC. <br /><br />**Note** {{prodname}} does not store registry credentials in its database and does not pull customer images into the {{prodname}} control plane. {{prodname}} uses Google’s encryption at rest feature to encrypt data across the entire datastore. |                                                              |
+| Kubernetes API server          | A Kubernetes component that validates and configures data for the API objects (for example, pods, services, and others). | &bull; TCP 6443 (from all components)                        |
+| kube-controllers               | Monitors the Kubernetes API and performs actions based on cluster state. The {{prodname}} kube-controllers container includes these controllers:<br/><br />&bull; Node<br />&bull; Service<br />&bull; Federated services<br />&bull; Authorization<br />&bull; Elasticsearch configuration<br />&bull; Managed cluster | &bull;  TCP 9094 from Prometheus API service<br />&bull;  TCP 6443 to Kubernetes API server |
```

Comment:
We should remove `Elasticsearch configuration` and `Managed cluster` because they don't run in a managed cluster.

Reply:
Removed.